perf/impl/dotnet-libp2p/v1.0/Program.cs

```diff
@@ -0,0 +1,313 @@
+                    // Get actual transfer amounts from the protocol instance
+                    // Since the protocol execution may have partially succeeded
+                    actualUploadBytes = PerfProtocol.ActualBytesSent;
+                    actualDownloadBytes = PerfProtocol.ActualBytesReceived;
+                    
+                    // Wait a moment to ensure completion and flush any output
+                    await Task.Delay(1000);
+                }
+                catch (Exception ex)
+                {
+                    logger.LogError("Protocol execution failed: {Message}", ex.Message);
+                    logger.LogError("Stack trace: {StackTrace}", ex.StackTrace);
+                    Environment.Exit(1);
+                }
+                finally
+                {
+                    try
+                    {
+                        await remotePeer.DisconnectAsync();
+                    }
+                    catch (Exception)
+                    {
+                    }
+                }
+
+                // Output final result like Go implementation
+                var elapsed = DateTime.UtcNow - startTime;
+                var result = new Result
+                {
+                    Type = "final",
+                    TimeSeconds = Math.Round(elapsed.TotalSeconds, 3),
+                    UploadBytes = actualUploadBytes,
+                    DownloadBytes = actualDownloadBytes
+                };
+
+                var jsonOutput = JsonSerializer.Serialize(result, new JsonSerializerOptions 
+                { 
+                    WriteIndented = false,
+                    PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+                });
+                
+                logger.LogCritical("{JsonOutput}", jsonOutput);
+            }
+            catch (Exception ex)
+            {
+                logger.LogError("Error: {Message}", ex.Message);
+                logger.LogError("Stack trace: {StackTrace}", ex.StackTrace);
+                
+                
+                var result = new Result
+                {
+                    Type = "final",
+                    TimeSeconds = 0.0,
+                    UploadBytes = 0UL,
+                    DownloadBytes = 0UL
+                };
+
+                var jsonOutput = JsonSerializer.Serialize(result, new JsonSerializerOptions 
+                { 
+                    WriteIndented = false,
+                    PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+                });
+                logger.LogInformation("{JsonOutput}", jsonOutput);
+                
+                Environment.Exit(1);
+            }
+        }
+    }
+    private static string GenerateDeterministicPeerId()
+    {
+        var fixedSeed = new byte[32];
+        Array.Fill(fixedSeed, (byte)0);
+        
+        var identity = new Identity(fixedSeed);
+       
+        return "12D3KooWBXu3uGPMkjjxViK6autSnFH5QaKJgTwW8CaSxYSD6yYL";
```

Comment:
Identity is generated and then ignored. Could you also confirm blank private key is not the same that go uses